docs/api/event/event.go

```diff
@@ -0,0 +1,158 @@
+package event
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	pollevent "sigs.k8s.io/cli-utils/pkg/kstatus/polling/event"
+	"sigs.k8s.io/cli-utils/pkg/object"
+)
+
+// Event describes something that has happened during the process or either
+// applying resources or removing resources from the cluster.
+// The Type property defines which event has taken place and the
+// corresponding property will contain more information about the details.
+type Event struct {
```

Comment:
Does `Event` need to have a field for runtime.Object or at least for the group, kind, name, namespace?